src/server/dflycmd.cc

```diff
@@ -390,10 +403,15 @@ void DflyCmd::StartStable(CmdArgList args, Transaction* tx, RedisReplyBuilder* r
     auto cb = [this, &status, replica_ptr = replica_ptr](EngineShard* shard) {
       FlowInfo* flow = &replica_ptr->flows[shard->shard_id()];
 
-      status = StopFullSyncInThread(flow, &replica_ptr->exec_st, shard);
-      if (*status != OpStatus::OK) {
-        return;
+      // We are doing partial sync. We never started FullSync so we don't need to stop it.
+      bool is_partial = flow->start_partial_sync_at.has_value();
+      if (!is_partial) {
+        status = StopFullSyncInThread(flow, &replica_ptr->exec_st, shard);
```

Comment:
what happens when if `!is_partial` ?
1. you run `StopFullSyncInThread`  for that thread only
2. then you run `StartStableSyncInThread` ? 

how this flow works?